tests/Fixer/Alias/ModernizeStrposFixerTest.php

```diff
@@ -40,86 +40,187 @@ public static function provideFixCases(): iterable
             '<?php if (0 === strpos($haystack1, $needle)) {}',
         ];
 
+        yield 'case insensitive yoda ===' => [
+            '<?php if (  str_starts_with(strtolower($haystack1), strtolower($needle))) {}',
+            '<?php if (0 === stripos($haystack1, $needle)) {}',
+        ];
+
         yield 'not zero yoda !==' => [
             '<?php if (  !str_starts_with($haystack2, $needle)) {}',
             '<?php if (0 !== strpos($haystack2, $needle)) {}',
         ];
 
+        yield 'case insensitive not zero yoda !==' => [
+            '<?php if (  !str_starts_with(strtolower($haystack2), strtolower($needle))) {}',
+            '<?php if (0 !== stripos($haystack2, $needle)) {}',
+        ];
+
         yield 'false yoda ===' => [
             '<?php if (  !str_contains($haystack, $needle)) {}',
             '<?php if (false === strpos($haystack, $needle)) {}',
         ];
 
+        yield 'case insensitive false yoda ===' => [
+            '<?php if (  !str_contains(strtolower($haystack), strtolower($needle))) {}',
+            '<?php if (false === stripos($haystack, $needle)) {}',
+        ];
+
         yield [
             '<?php if (str_starts_with($haystack3, $needle)  ) {}',
             '<?php if (strpos($haystack3, $needle) === 0) {}',
         ];
 
+        yield [
+            '<?php if (str_starts_with(strtolower($haystack3), strtolower($needle))  ) {}',
+            '<?php if (stripos($haystack3, $needle) === 0) {}',
+        ];
+
         yield 'casing call' => [
             '<?php if (str_starts_with($haystack4, $needle)  ) {}',
             '<?php if (STRPOS($haystack4, $needle) === 0) {}',
         ];
 
+        yield 'case insensitive casing call' => [
+            '<?php if (str_starts_with(strtolower($haystack4), strtolower($needle))  ) {}',
+            '<?php if (STRIPOS($haystack4, $needle) === 0) {}',
+        ];
+
         yield 'leading namespace' => [
             '<?php if (\str_starts_with($haystack5, $needle)  ) {}',
             '<?php if (\strpos($haystack5, $needle) === 0) {}',
         ];
 
+        yield 'case insensitive leading namespace' => [
+            '<?php if (\str_starts_with(\strtolower($haystack5), \strtolower($needle))  ) {}',
```

Comment:
I've added namespaces to strtolower because str_starts_with has it. Does that make sense?

Reply:
i think we can have good reasons for both options, ok to stay with your suggestion